<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>720</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>通讯配置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>端口:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="zigbee_serialportBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="zigbee_baudrateBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="currentText">
      <string>9600</string>
     </property>
     <item>
      <property name="text">
       <string>1382400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>921600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>76800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>43000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="zigbee_openButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>105</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>320</y>
      <width>211</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>任务</string>
    </property>
    <widget class="QPushButton" name="newTaskButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>新增任务</string>
     </property>
    </widget>
    <widget class="QComboBox" name="taskBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>89</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="postTaskButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>发布任务</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>终点:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> Y:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editEndX">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="editEndY">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteTaskButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>删除任务</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>任务:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editID">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>570</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>地图</string>
    </property>
    <widget class="QPushButton" name="updateMapButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>更新坐标轴</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>地图大小(cm)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>62</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>62</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMapX">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>60</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMapY">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>60</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>定位模块</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>端口:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="uwb_serialportBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="uwb_baudrateBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="currentText">
      <string>115200</string>
     </property>
     <item>
      <property name="text">
       <string>1382400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>921600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>76800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>43000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="uwb_openButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>105</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
